Makefile.toml

```diff
@@ -397,6 +397,9 @@ script = '''
 
 set -e
 
+
+echo "$(tput bold)$(tput setaf 9)RisingWave playground SHOULD NEVER be used in benchmarks and production environment!!!
```

Comment:
shall we move this to `risingwave_cmd_all`? Users can also start playground by using that binary.

Reply:
I had also thought about it. The benefit of putting it here is that it is before compiling and running, so it is very noticable without being mixed in logs... 🤪

What about putting it at both places?

Reply:
Sounds good.

Reply:
Moved to cmd & added a `sleep` to make it more noticable. Is that OK?

<img width="1140" alt="image" src="https://user-images.githubusercontent.com/37948597/203848834-e1facdc1-5553-4726-ba99-b0edb0babd93.png">